core/scripts/ocr2dr/templates/bootstrap.toml

```diff
@@ -0,0 +1,10 @@
+type = "bootstrap"
+schemaVersion = 1
+name = "OCR2DR bootstrap"
```

Comment:
Maybe add some unique id or timestamp to job names?

Reply:
Done